python/pyarrow/tests/test_fs.py

```diff
@@ -1052,6 +1052,50 @@ def test_s3_options(monkeypatch):
         )
 
 
+@pytest.mark.s3
+def test_s3_proxy_options(monkeypatch):
+    from pyarrow.fs import S3FileSystem
+
+    # Avoid wait for unavailable metadata server in ARN role example below
+    monkeypatch.setenv("AWS_EC2_METADATA_DISABLED", "true")
```

Comment:
Doesn't seem useful? There's no ARN role example below :-)

Reply:
My bad, I just copied it over from the test above. Will remove.